### PR TITLE
updated-supportcfg-and-topic-map

### PR DESCRIPTION
**Pull request by ssugandh**
The issue for genearting microspecs was in the topic_map file. One of the file names that I added was wrong, I corrected. 
Thanks for the tip Ian to point me to look into directives.

### CODE REVIEW

**Comment by ssugandh** [comment]
Applicable for PCSUP-7947, PCSUP-8061, and GitHub (Twistlock) issue 36317

File: api/descriptions/custom-rules/custom-rules.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-Custom runtime rules define precise runtime defense actions for containers, hosts, and Kubernetes audits.
+Defines a precise defense action for containers, hosts, and app-embedded audits.
```
Comment (iansk) [inline api/descriptions/custom-rules/custom-rules.md]: @ssugandh Revert back to "Defines precise runtime defense actions for containers, hosts, and Kubernetes audits."

"Kubernetes audits" is a product feature.

Comment (ssugandh) [reply api/descriptions/custom-rules/custom-rules.md]: @iansk : I reverted. I corrected due to the change in the UI.
image

Comment (iansk) [reply api/descriptions/custom-rules/custom-rules.md]: @ssugandh Good point. So we actually need to extend the description to include WAAS requests and WAAS responses. Also, we spell "Kubernetes" with a capital "K". Please update.

Comment (iansk) [reply api/descriptions/custom-rules/custom-rules.md]: @ssugandh What about WAAS custom rules? This endpoint is also used for that

File: api/descriptions/custom-rules/custom-rules.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
-Custom runtime rules define precise runtime defense actions for containers, hosts, and Kubernetes audits.
+Defines a precise defense action for containers, hosts, and Kubernetes audits.
 Custom rules can be individually added to runtime and WAAS policies.
```
Comment (ssugandh) [inline api/descriptions/custom-rules/custom-rules.md]: @iansk : The text about WAAS policies is already there.

Comment (iansk) [reply api/descriptions/custom-rules/custom-rules.md]: @ssugandh Let's just change line 1 to this:

`Defines a precise runtime defense action for containers, hosts, Kubernetes audits, WAAS requests, and WAAS responses.`

And then go ahead and remove the second line.

**Review by iansk — CHANGES_REQUESTED** [review]

**Review by iansk — APPROVED** [review]

**Pull request merged**